src/onedb/onedb_live.rb

```diff
@@ -96,6 +97,19 @@ def select(table, where)
         [row].flatten.compact
     end
 
+    def count(table, where)
+        sql = "SELECT COUNT(*) FROM #{table} WHERE #{where}"
+        res = db_query(sql, 'Error querying databse')
```

Comment:
small typo: database

Reply:
Ups, that function shouldn't be there, thanks @tinova for noticing that!